theolive/getting-started/authorization.mdx

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 1
+sidebar_label: Authorization
+pagination_label: Authorization
+pagination_next: getting-started/regions
+title: Autorization
+---
+
+We make sure that only authorized users can make API calls!
+
+---
+
+Making secure calls to our REST API requires authentication and authorization. THEOlive makes use of token-secret pairs to authorize requests.
+
+## 1. Getting a token-secret pair
+
+To generate a token-secret pair, you should be registered for an account at on our [management console](https://console.theo.live/). Under the section "Tokens", click on the "Generate Token" button, and enter a name before clicking "Generate".
+
+![](../assets/img/c0aca77-1.-THEOlive-console-generate-token.PNG)
+
+This will give a pop-up which shows your token "key" and the corresponding secret.
+
+![](../assets/img/bb9b4a6-2.-THEOlive-console-generated-token-and-secret.PNG)
+
+Once you "Close" the pop-up window, the new token will be visible in your list of tokens.
+
+![](../assets/img/5361242-3.-THEOlive-console-token-list.PNG)
+
+> 🚧 Please store the secret
+>
+> Make sure to store the secret somewhere, as this will be shown to you only once, and THEOlive only stores a hash of the secret, so it cannot be recovered afterward.
```

Comment:
Pro tip: try using [admonitions](https://docusaurus.io/docs/markdown-features/admonitions) to make these prettier. 😁 

For example:
```suggestion
:::warning[Please store the secret]

Make sure to store the secret somewhere, as this will be shown to you only once, and THEOlive only stores a hash of the secret, so it cannot be recovered afterward.

:::
```